presto-hive/src/main/java/io/prestosql/plugin/hive/util/HiveBucketing.java

```diff
@@ -241,19 +245,19 @@ static int getBucketNumber(int hashCode, int bucketCount)
         return Optional.of(new HiveBucketFilter(builder.build()));
     }
 
-    private static Optional<Set<Integer>> getHiveBuckets(Table table, Map<ColumnHandle, List<NullableValue>> bindings)
+    private static Optional<Set<Integer>> getHiveBuckets(HiveBucketProperty hiveBucketProperty, HiveTableHandle hiveTable, Map<ColumnHandle, List<NullableValue>> bindings)
```

Comment:
Changed to pass `HiveBucketProperty hiveBucketProperty, List<Column> dataColumns` instead of `HiveBucketProperty hiveBucketProperty, HiveTableHandle hiveTable`